src/stores/coreSettings.ts

```diff
@@ -250,6 +250,13 @@ export const CORE_SETTINGS: SettingParams[] = [
     type: 'boolean',
     defaultValue: false
   },
+  {
```

Comment:
What is the definition of title here? If I am the user I would be confused on what title means for a file.

Reply:
added a tooltip to make it clearer

Reply:
I still find that `configurable model metadata title` term confusing. Can you just put `Excludes the directory and the '.safetensors' file extension` in the simplified name doc string here?